frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -263,11 +263,22 @@ function stringToBoolean(s) {
     return s == "true";
 }
 
+function truncateText(text, length) {
+    if (text) {
+        if (text.length > length) {
+            return text.substring(0, length) + "...";
```

Comment:
why not `text.substring(0, length - 3) + "...";`

Reply:
removed function `truncateText`